templates/contact-us.html

```diff
@@ -7,15 +7,22 @@
   <section class="p-strip">
     <div class="row">
       <div class="col-8">
-        <h1>Contact us about MicroK8s</h1>
-        <p class="p-heading--four">Fill in your details below and a member of our team will be in touch.</p>
+        <h1>Talk to our Kubernetes experts</h1>
       </div>
     </div>
     <div class="row">
       <div class="col-8">
         <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_3230">
           <fieldset class="u-no-margin--bottom">
-            <h3>About you</h3>
+            <h3>Let us know about your Kubernetes use case</h3>
+            <ul class="p-list">
+              <li class="mktFormReq mktField p-list__item">
+                <textarea required id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" class="mktoField mktoRequired" maxlength="2000" placeholder="Anything you'd like to communicate about your needs or interests?"></textarea>
```

Comment:
```suggestion
                <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" class="mktoField mktoRequired" maxlength="2000" placeholder="Anything you'd like to communicate about your needs or interests?"></textarea>
```

Reply:
Can you remove this required please? Checked [here](https://docs.google.com/document/d/1KkLHMrGipY4zgIXreujpyJlLvYUs-iAlP-FtpYrZmWA/edit?disco=AAAAPnJOdSg)